homeassistant/components/sensor/discogs.py

```diff
@@ -6,11 +6,12 @@
 """
 from datetime import timedelta
 import logging
+import random
 
 import voluptuous as vol
 
 from homeassistant.components.sensor import PLATFORM_SCHEMA
-from homeassistant.const import ATTR_ATTRIBUTION, CONF_NAME, CONF_TOKEN
+from homeassistant.const import ATTR_ATTRIBUTION, CONF_TOKEN
```

Comment:
Removed the name sensor because this sensor will now generate:

```
sensor.discogs_collection
sensor.discogs_wantlist
sensor.discogs_random_record
```

instead of the previous: `sensor.discogs`